include/boost/gil/image_processing/filter.hpp

```diff
@@ -60,6 +65,79 @@ void blur(
     box_filter(src_view, dst_view, kernel_size, anchor, true, option);
 }
 
+
+namespace detail
+{
+template <typename SrcView, typename DstView>
+void impl_median(
+    SrcView const& src_view,
+    DstView const& dst_view,
+    std::size_t kernel_size
+)
```

Comment:
Nitpicking for future: this breaking is unnecessary as it could fit our 100 characters line length limit

```cpp
void impl_median(SrcView const& src_view, DstView const& dst_view, std::size_t kernel_size)
```